src/webrtc/Server.ts

```diff
@@ -48,18 +49,18 @@ export class Server {
 			});
 		}
 
-		// this.server.on("upgrade", (request, socket, head) => {
-		// 	if (!request.url?.includes("voice")) return;
-		// 	this.ws.handleUpgrade(request, socket, head, (socket) => {
-		// 		// @ts-ignore
-		// 		socket.server = this;
-		// 		this.ws.emit("connection", socket, request);
-		// 	});
-		// });
+		this.server.on("upgrade", (request, socket, head) => {
+			if (!request.url?.includes("voice")) return;
+			this.ws.handleUpgrade(request, socket, head, (socket) => {
+				// @ts-ignore
+				socket.server = this;
+				this.ws.emit("connection", socket, request);
+			});
+		});
 
 		this.ws = new ws.Server({
 			maxPayload: 1024 * 1024 * 100,
-			server: this.server,
+			noServer: true,
```

Comment:
???

Reply:
Rory&, this was done so that the websocket server would not be responsible for upgrading http connections, which is now done above with the uncommented update listener. This was done so that voice ws is mounted on /voice

I don't think this is necessary though, I would rather it just use a different port? This makes routing through nginx less difficult when using bundle.